# Control parameters
max_vel: 0.38
min_vel: 0.05
max_turn_rate: 0.7
xy_goal_tolerance: 0.01
yaw_goal_tolerance: 0.2
controller_hz: 100.0
# soon this will be changed to the local path solution
control_path_topic: /smf_move_base_planner/smf_move_base_solution_path
control_output_topic: /pepper/cmd_vel
odometry_topic: /pepper/odom_groundtruth
